lib/colorado_lottery.rb

```diff
@@ -0,0 +1,53 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.age >= 18 && contestant.game_interests.include?(game.name)
+  end
+
+  def out_of_state_only_for_national(contestant, game)
+    if game.national_drawing? == false
+      contestant.out_of_state? == false
+    else
+      true
+    end
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) && out_of_state_only_for_national(contestant, game)
+  end
+
+  def register_contestant(contestant, game)
+    if can_register?(contestant, game) == true
```

Comment:
This is slightly redundant, because the method returns a boolean which will be true or false. We never need to ask `true == true` which is what this boils down to. But, I do like that you have this condition. Even though the interaction pattern only ever asks to register contestants that can register. This is a nice guard to ensure that this method works in conjunction with `can_register?`